src/System.Runtime.Extensions/tests/System/Diagnostics/Stopwatch.cs

```diff
@@ -93,6 +94,35 @@ public static void StartNewAndRestart()
             }
         }
 
+        [OuterLoop("Sleeps for relatively long periods of time")]
+        [Fact]
+        public static void ElapsedMilliseconds_WithinExpectedWindow()
+        {
+            const int AllowedTries = 30;
+            const int SleepTime = 1000;
+            const double WindowFactor = 2;
+
+            var results = new List<long>();
+
+            var sw = new Stopwatch();
+            for (int trial = 0; trial < AllowedTries; trial++)
+            {
+                sw.Restart();
+                Thread.Sleep(SleepTime);
+                sw.Stop();
+
+                if (sw.ElapsedMilliseconds >= (SleepTime / WindowFactor) &&
+                    sw.ElapsedMilliseconds <= (SleepTime * WindowFactor))
+                {
+                    return;
+                }
+
+                results.Add(sw.ElapsedMilliseconds);
+            }
+
+            Assert.True(false, $"All {AllowedTries} fell outside of {WindowFactor} window of {SleepTime} sleep time: {string.Join(", ", results)}");
```

Comment:
Were you able to validate that this failed without https://github.com/dotnet/corefx/pull/37236?

Reply:
Yes.  I reverted that commit in my WSL install and got this failure:
```
      System.Diagnostics.Tests.StopwatchTests.ElapsedMilliseconds_WithinExpectedWindow [FAIL]
        All 30 fell outside of 2 window of 1000 sleep time: 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10
        Expected: True
        Actual:   False
```